newrelic/exporter.go

```diff
@@ -53,34 +63,109 @@ func NewExporter(serviceName, apiKey string, options ...func(*telemetry.Config))
 	}
 	return &Exporter{
 		harvester:   h,
-		serviceName: serviceName,
+		serviceName: service,
 	}, nil
 }
 
+// NewExportPipeline creates a new OpenTelemetry telemetry pipeline using a
+// New Relic Exporter configured with default setting. It is the callers
+// responsibility to stop the returned push Controller. This function uses the
+// following environment variables to configure the exporter installed in the
+// pipeline:
+//
+//    * `NEW_RELIC_API_KEY`: New Relic API key.
+//    * `NEW_RELIC_METRIC_URL`: URL to New Relic metric endpoint.
+//    * `NEW_RELIC_METRIC_URL`: URL to New Relic trace endpoint.
```

Comment:
Could we add either an explanation of how to find these endpoints, or a link to where they can be found?

Reply:
Updated.